modules/core/src/test/kotlin/datamaintain/FilterTest.kt

```diff
@@ -166,4 +166,45 @@ internal class FilterTest {
             get(0).get { this.name }.isEqualTo("01_file1")
         }
     }
+
+    @Test
+    fun `should filter scripts by name`() {
+        // Given
+        val filenameRegex = Regex("(.*)(file1$)")
```

Comment:
The groups here are not necessary and I find them confusing
```suggestion
        val filenameRegex = Regex(".*file1$")
```